service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -76,7 +80,11 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
 
         Document document = uploadDocumentService.uploadPDF(userId, authorization, pdf, buildFileName(caseDetails));
 
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+        sdf.setTimeZone(TimeZone.getTimeZone("Europe/London"));
```

Comment:
Isn't taking default time zone good enough?

Reply:
I seem to remember an issue due to it selecting UTC by default, but we are currently in BST...

Reply:
yeah but funny bst actually sets it to asia time if you check the mappings of the documentation and so europe/london was correct for the time zone we are in.